ios/brave-ios/Sources/AIChat/Components/Settings/AIChatAdvancedSettingsLabelDetailView.swift

```diff
@@ -6,7 +6,6 @@
 import BraveUI
 import SwiftUI
 
-@available(iOS, introduced: 15.0, deprecated: 16.0)
```

Comment:
@Brandon-T this did not have an iOS 16+ counterpart so just removed the deprecation notice entirely